tomviz/CMakeLists.txt

```diff
@@ -558,6 +558,7 @@ target_link_libraries(tomvizlib
     vtkjsoncpp
     vtkpugixml
     tomvizExtensions
+    h5cpp
```

Comment:
In Tomviz we should just be able to add the C++ source files to the main build. I would like to avoid adding an extra library for this.